waku/v2/protocol/filter/filter_proto_ident_test.go

```diff
@@ -225,7 +225,7 @@ func (s *FilterTestSuite) TestIncorrectSubscribeIdentifier() {
 	s.ctx, s.ctxCancel = context.WithTimeout(context.Background(), 10*time.Second) // Test can't exceed 10 seconds
 
 	s.testTopic = "/waku/2/go/filter/test"
-	s.testContentTopic = "TopicA"
+	s.testContentTopic = "/test/10/my-app"
```

Comment:
Better to use a const instead of copy pasting the same value in multiple places which is error prone.

Reply:
I agree, fixed at [8c755ff](https://github.com/waku-org/go-waku/pull/916/commits/8c755ffc1ffcc70142b5f151899cf94d540bbd0a)